raiden_contracts/deploy/contract_deployer.py

```diff
@@ -151,7 +151,8 @@ def deploy_raiden_contracts(
             "contracts": {},
         }
 
-        self._deploy_and_remember(CONTRACT_ENDPOINT_REGISTRY, [], deployed_contracts)
+        if self.contract_manager.has_contract(CONTRACT_ENDPOINT_REGISTRY):
```

Comment:
Unless I'm misunderstanding this can't happen since the source has been removed?

Reply:
The deployer can still deploy the RedEyes version.  Maybe we can drop it.  https://github.com/raiden-network/raiden-contracts/issues/1057

Reply:
Ah, thanks. I didn't see that this can also work on the pre-compiled json data.